include/clap/events.h

```diff
@@ -113,6 +113,10 @@ enum {
    CLAP_EVENT_MIDI,       // raw midi event; clap_event_midi
    CLAP_EVENT_MIDI_SYSEX, // raw midi sysex event; clap_event_midi_sysex
    CLAP_EVENT_MIDI2,      // raw midi 2 event; clap_event_midi2
+
+   // Represents a trigger
+   // Uses clap_event_trigger.
+   CLAP_EVENT_TRIGGER,
```

Comment:
Use the same style as above for the description please

Reply:
What do you mean with 'style' in this case? Language or punctuation at the end of the line?

Reply:
Comment at the end of the line.
But actually forget what I said, as we use both styles. Just that I was just looking at the 3 line above in the github review thing, my bad.